### PR TITLE
refactor: add profileInfo from server

### PR DESCRIPTION
**Pull request by EvgeniyaYakunina**
### **Refactor Profile Page & Implement Persistent User Data**

**Description:**

This pull request introduces two significant changes to improve the profile page functionality and enhance user experience:

**Separate Profile Picture Component:**

Created a new reusable component ProfilePicture to handle the user's profile picture.
Moved the existing profile picture logic into this new component.
Benefits: Increased code reusability, easier maintenance, and improved separation of concerns.

**Implement Persistent User Data:**

Added functionality to fetch and store user data from the server.
Updated the profile page to display user information using the fetched data.
Implemented local storage to cache user data, ensuring it persists across sessions.
Benefits: Improved user experience by reducing server requests, faster load times, and better data consistency.

### CODE REVIEW

**Review by Emilo067 — APPROVED** [review]

File: src/features/profile/ui/profile/profile.tsx [hunk 1]
```diff
@@ -1,11 +1,10 @@
 import { ComponentPropsWithoutRef } from 'react'
 
 import { PostList } from '@/features'
-import { avaTest } from '@/shared/assets'
 import { ProfileStat } from '@/shared/ui'
+import { ProfileAvatar } from '@/shared/ui/profileAvatar'
```
Comment (Lelik7777) [inline src/features/profile/ui/profile/profile.tsx]: update import

File: src/shared/ui/profileAvatar/profileAvatar.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { avaTest } from '@/shared/assets'
+import clsx from 'clsx'
+import Image from 'next/image'
+
+import styles from './profileAvatar.module.scss'
+
+type Props = {
```
Comment (Lelik7777) [inline src/shared/ui/profileAvatar/profileAvatar.tsx]: ProfileAvatarProps

File: src/shared/ui/profileAvatar/profileAvatar.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  avatarOwner: string | undefined
+  className?: string
+  height?: number
+  width?: number
+}
+export const ProfileAvatar = (props: Props) => {
```
Comment (Lelik7777) [inline src/shared/ui/profileAvatar/profileAvatar.tsx]: здесь реализовать деструктуризацию

**Comment by Lelik7777** [comment]
Add short descritption for PR

**Pull request merged**